doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
++=====================================================+==========================================================================================================+
+| ``matter_weather_station/zephyr/zephyr.signed.bin`` | Signed (with MCUboot signing key) main application binary file which can be used directly with MCUmgr    |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``signed_by_mcuboot_and_b0_ipc_radio.bin``          | Signed (with b0 and MCUboot signing key) network core IPC radio update from application core binary file |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``dfu_multi_image.bin``                             | DFU multi image file containing firmware update files and manifest                                       |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``matter.ota``                                      | Matter over-the-air firmware update file                                                                 |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``dfu_application.zip``                             | Zip file containing firmware update files and manifest                                                   |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+
+.. _child_parent_to_sysbuild_migration_image_overlay_changes:
+
+Image overlay configuration
+###########################
+
+In child/parent image, an application could include additional configuration files in the ``child_image`` folder that would be applied to these images (see :ref:`ug_multi_image_permanent_changes`), this feature has been adapted in sysbuild, see :ref:`sysbuild_application_configuration` for an overview. Child/parent image configuration must be updated to be be usable from sysbuild, the way these files can be used differs. In child/parent image, there can be Kconfig fragments and board overlays which are all used, in sysbuild this is not supported, there can either be a Kconfig fragment overlay, or the whole application configuration directory can be changed. If an image application configuration directory is created then it must include all the required files for that image as none of the original application configuration files will be used. Sysbuild includes support for :ref:`application-file-suffixes` in applications and :ref:`sysbuild_file_suffixes` can also be used.
```

Comment:
```suggestion
In child/parent image configurations, an application could include additional configuration files in the ``child_image`` folder that would be applied to these images (see :ref:`ug_multi_image_permanent_changes`).
This feature has been adapted in Sysbuild; see :ref:`sysbuild_application_configuration` for an overview. 

You must update child/parent image configuration to use it with Sysbuild, as the way these files can be used differs:

* In child/parent image configurations, there can be Kconfig fragments and board overlays that are all used.
* In Sysbuild, this is not supported.
  You can either have a Kconfig fragment overlay or change the whole application configuration directory. 

If an image application configuration directory is created, it must include all the required files for that image, as none of the original application configuration files will be used.
Sysbuild includes support for :ref:`application-file-suffixes` in applications, and it can also use :ref:`sysbuild_file_suffixes`.
```